overrides/scripts/ExtraUtilities.zs

```diff
@@ -0,0 +1,422 @@
+import scripts.CommonVars.makeShaped as makeShaped;
+import scripts.CommonVars.makeExtremeRecipe5 as makeExtremeRecipe5;
+
+//XU2 Machine Block
+recipes.removeByRecipeName("extrautils2:machine_base");
+makeShaped("machine_block",
+	<extrautils2:machine>, [
+		"SSS",
+		"SQS",
+		"SSS",
+	], {
+		E: <metaitem:plateSteel>,
+		C: <ore:gemQuartzBlack>
+	});
```

Comment:
Juan (superbcow#4144) pointed out on Discord that the replacement mappings used here don't match the characters used in the recipe.

Reply:
Fixed